pkg/controllers/dynakube/extension/eec/statefulset.go

```diff
@@ -56,11 +59,14 @@ const (
 	activeGateTrustedCertMountPath     = "/var/lib/dynatrace/secrets/ag"
 	activeGateTrustedCertSecretKeyPath = "server.crt"
 	httpsCertVolumeName                = "https-certs"
-	httpsCertFile                      = "cert.pem"
-	httpsPrivKeyFile                   = "key.pem"
 	httpsCertMountPath                 = "/var/lib/dynatrace/remotepluginmodule/secrets/https/"
```

Comment:
The last `/` leads to double dashes in the full file path later on, plz. remove it while you're at it.

Reply:
fixed in bc76d466